src/supercrafter333/BetterBan/EventListener.php

```diff
@@ -168,5 +168,12 @@ public function onLogin(PlayerLoginEvent $event) : void {
 			$player->kick($reason);
 			$event->cancel();
 		}
+		if (BetterBan::isBannedIp($player->getNetworkSession()->getIp())) {
```

Comment:
PLease use proper indentation. Please fix your coding standards as follows to poggit rules.